numba/cuda/cudadrv/nvvm.py

```diff
@@ -400,17 +400,17 @@ def get(self):
 
 
 ir_numba_atomic_max = """
-define internal {T} @___numba_atomic_{T}_max({T}* %ptr, {T} %val) alwaysinline {{
+define internal {T} @___numba_atomic_{T}_{NAN}max({T}* %ptr, {T} %val) alwaysinline {{
```

Comment:
I think the two templates for max/min can be unified following the latest changes?